sci-biology/STAR/STAR-2.5.3a.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="STAR aligner: align RNA-seq reads to reference genome uncompressed suffix arrays"
+HOMEPAGE="http://code.google.com/p/rna-star/"
+SRC_URI="https://github.com/alexdobin/STAR/archive/"${PV}".tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+# sci-biology/htslib
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+# contains bundled STAR-Fusion
+src_prepare(){
+	sed -e "s/= gcc/= $(tc-getCC)/;s/-O2/${CFLAGS}/g" -i source/htslib/Makefile || die
+	sed -e "s/-O3/${CFLAGS}/g" -i source/Makefile || die
+}
```

Comment:
I strongly doubt you even tested this ebuild...

Reply:
The ebuild is functional, I have it installed on my system. The commits, however, were only a EAPI and version bump (as referred to in the commit message), I did not check whether these lines are required or even functional.

If this is something I should have investigated more carefully, please write this during review and do not assume I haven't tested it.

Reply:
```
$ ebuild STAR-2.5.3a.ebuild compile
[...]
 * ERROR: sci-biology/STAR-2.5.3a::science failed (prepare phase):
 *   eapply_user (or default) must be called in src_prepare()!
 * 
 * Call stack:
 *            ebuild.sh, line  776:  Called __ebuild_main 'prepare'
 *   phase-functions.sh, line 1053:  Called __dyn_prepare
 *   phase-functions.sh, line  383:  Called die
 * The specific snippet of code:
 *   		die "eapply_user (or default) must be called in src_prepare()!"
 * 
 * If you need support, post the output of `emerge --info '=sci-biology/STAR-2.5.3a::science'`,
 * the complete build log and the output of `emerge -pqv '=sci-biology/STAR-2.5.3a::science'`.
 * The complete build log is located at '/var/tmp/portage/sci-biology/STAR-2.5.3a/temp/build.log'.
 * The ebuild environment file is located at '/var/tmp/portage/sci-biology/STAR-2.5.3a/temp/environment'.
 * Working directory: '/var/tmp/portage/sci-biology/STAR-2.5.3a/work/STAR-2.5.3a'
 * S: '/var/tmp/portage/sci-biology/STAR-2.5.3a/work/STAR-2.5.3a'
```

Reply:
In general it goes without saying that if you add a new ebuild to the system, you have at least done the basic `ebuild ... install` routines to check that the basics of it works.

Reply:
Yes, you're right, this now fails. - I'm guessing this check has been added to portage more recently than my PR? (and prefix portage at the time).

I'll fix it before resubmitting. Thank you for spotting this.

Reply:
Nope, that `eapply_user()` check has been there for ages (since EAPI 6 support was integrated into Portage).